pkg/cmd/authLogin_test.go

```diff
@@ -24,3 +26,62 @@ func TestAuthLoginCommandInCommandCollection(t *testing.T) {
 	assert.IsType(t, &AuthLoginCmdValues{}, authCommand.Values())
 	assert.NotNil(t, authCommand.CobraCommand())
 }
+
+func TestAuthLoginHelpFlagSetCorrectly(t *testing.T) {
+	// Given...
+	factory := NewMockFactory()
+
+	// Note: No --maven or --gradle flags here:
+	var args []string = []string{"auth", "login", "--help"}
+
+	// When...
+	err := Execute(factory, args)
+
+	// Then...
+
+	// Check what the user saw is reasonable.
+	stdOutConsole := factory.GetStdOutConsole().(*utils.MockConsole)
+	outText := stdOutConsole.ReadText()
+	assert.Contains(t, outText, "Displays the options for the 'auth login' command.")
+
+	stdErrConsole := factory.GetStdErrConsole().(*utils.MockConsole)
+	errText := stdErrConsole.ReadText()
+	assert.Empty(t, errText)
```

Comment:
These 3 sections are repeated many times. Perhaps a method like 
checkOutput(expectedStdOut, expectedStdErr, expectedFinalError) would compress this test code ?

Reply:
Or even testCommandCombination(args, expectedExecuteError, expectedStdOut, expectedStdErr, expectedFinalError)

Reply:
made a function for the creation of the command collection and replacing of the the runE function as well.